src/popup/components/TransactionList.vue

```diff
@@ -189,6 +207,13 @@ export default {
   display: flex;
   flex-direction: column;
 
+  .search-bar-wrapper {
+    position: sticky;
+    top: 126px;
```

Comment:
Magic number. Also I think this should be controlled by the parent component as this one is generic one and can be reused within many places.